cilium-docker-plugin.Dockerfile

```diff
@@ -3,8 +3,7 @@ LABEL maintainer="maintainer@cilium.io"
 ADD . /go/src/github.com/cilium/cilium
 WORKDIR /go/src/github.com/cilium/cilium/plugins/cilium-docker
 ARG LOCKDEBUG
-ARG V
-RUN make CGO_ENABLED=0 GOOS=linux LOCKDEBUG=$LOCKDEBUG PKG_BUILD=1 EXTRA_GOBUILD_FLAGS="-a -installsuffix cgo"
+RUN make LOCKDEBUG=$LOCKDEBUG
```

Comment:
`V` is interpreted by make here to make the inner build quiet, isn't it?

(I haven't explicitly tested this recently, but I always define `V=0` in my environment)

Reply:
It would be, but it wasn't passed along, on that basis I'm removing it. Also, I believe we make it quite on the outside and there is no need to control it here.